pkg/testutils/cluster.go

```diff
@@ -0,0 +1,34 @@
+package testutils
+
+import (
+	"time"
+
+	"github.com/aws/aws-sdk-go/aws"
+	awseks "github.com/aws/aws-sdk-go/service/eks"
+)
+
+// NewFakeCluster creates a new fake cluster to be used in the tests
+func NewFakeCluster(clusterName string, status string) *awseks.Cluster {
+	created := &time.Time{}
+
+	cluster := &awseks.Cluster{
+		Name:      aws.String(clusterName),
+		Status:    aws.String(status),
+		Arn:       aws.String("arn-12345678"),
+		CreatedAt: created,
+		ResourcesVpcConfig: &awseks.VpcConfigResponse{
+			VpcId:     aws.String("vpc-1234"),
+			SubnetIds: []*string{aws.String("sub1"), aws.String("sub2")},
+		},
+	}
+
+	return cluster
+}
+
+// ValidClusterStatus checks that the provided status is a valid
+// status of a cluster according to AWS SDK
+func ValidClusterStatus(status string) bool {
+	valid := map[string]bool{awseks.ClusterStatusActive: true, awseks.ClusterStatusCreating: true, awseks.ClusterStatusDeleting: true, awseks.ClusterStatusFailed: true}
```

Comment:
I'd put each key on a separate line here..

Reply:
Actually, is this used anywhere yet?
I'm not entirely sure about having it here, I though we already have some filtering functionality in `pkg/eks/api.go`.. I'd try to fix whatever needs fixing there, e.g. pass filter option or something like that...

Reply:
Actually, it'd be good to leave any major improvements to how we get out cluster until after #132, as after that we would be looking at only CloudFormation-managed clusters. When we do that, we might want to let anyone list all EKS clusters with e.g. something like `--include-unmanaged` or `--incude-all`.

Reply:
I removed this function. I was going to call it from `NewFakeCluster` to valid the status being passed it. But as its a test i decided not to and left this in. Its been removed.